ci-operator/jobs/openstack-k8s-operators/openstack-baremetal-operator/openstack-k8s-operators-openstack-baremetal-operator-main-presubmits.yaml

```diff
@@ -0,0 +1,332 @@
+presubmits:
+  openstack-k8s-operators/openstack-baremetal-operator:
+  - agent: kubernetes
+    always_run: true
+    branches:
+    - ^main$
```

Comment:
since you updated the file under config directory, you need to re-generated this job file, with: `make jobs`

Reply:
thank you doing